packages/smooth_app/lib/pages/product/new_product_page.dart

```diff
@@ -54,6 +53,9 @@ class _ProductPageState extends State<ProductPage> {
     AnalyticsHelper.trackProductPageOpen(
       product: _product,
     );
+    knowledgePanels = KnowledgePanelsQuery(
```

Comment:
so this will reissue a backend query, just to confirm, knowledge_panels are not fetched as part of the product query when we scan a product ? 

Also, will this be cached ?

Reply:
@jasmeet0817 The answers are no and no. Good point, cf. https://github.com/openfoodfacts/smooth-app/pull/1666#issuecomment-1111013154